### PR TITLE
SITES-15210: Added Extension Manager docs

### PR DESCRIPTION
**Pull request by lenaorobei**
## Description

- added Extension Manager docs
- reorganized the pages for better discoverability

Preview -> https://developer-stage.adobe.com/uix/docs/

## Motivation and Context
Improve the overall user experience with UI extensibility documentation.

### CODE REVIEW

File: src/pages/extension-manager/index.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+| Can see the list of available extensions | ✅ | ✅ | ✅ |
+| Can enable/disable extensions | ❌ | ✅  | ✅ |
+| Can configure extensions | ❌ | ✅  | ✅ |
+| Can preview extensions | ✅ | ✅ | ✅ |
+| Can add BYO extensions | ❌ | ✅ | ✅ |
+
+## Extensions Listing
+Once you're on the [Extension Manager](https://experience.adobe.com/aem/extension-manager) home page, you can explore the list of AEM programs and environments available for your organization. Go ahead and select a program and an environment to display the available extensions.
+
+For a deep dive inot Extension Manager's capabilities, head to the [Feature Highlights](feature-highlights) section.
```
Comment (irenelagno) [inline src/pages/extension-manager/index.md:56]: For a deep dive **into** Extension Manager's capabilities, head to the [Feature Highlights](feature-highlights) section.

File: src/pages/index.md [hunk 2]
```diff
@@ -5,85 +5,110 @@ contributors:
...
 <DiscoverBlock slots="heading, link, text"/>
 
-### Publish
+## Experiment
 
-[UI Extensions Management Guide](guides/publication/)
+[Extension Manager in AEM Sites](extension-manager/)
 
-Know how to deploy and publish UI Extensions for your organization.
+Discover your UI Extensions within the safe playground and experiment with your AEM setup
```
Comment (irenelagno) [inline src/pages/index.md:111]: with your AEM Sites setup

**Comment by fe-lix-** [comment]
Should we create a redirect from `/overview` to `/getting-started`? I think this should be achievable with this: https://www.gatsbyjs.com/docs/how-to/cloud/working-with-redirects-and-rewrites/ 

It seems though no one in the AdobeDocs org has used `createRedirect` until now.

**Review by fe-lix- — APPROVED** [review]

**Comment by lenaorobei** [comment]
Hey @fe-lix-, looks like the redirects are handled by the Developer Website team via Jira tickets.

**Pull request merged**